cmd/install/assets/hypershift_operator.go

```diff
@@ -1217,13 +1218,15 @@ func HyperShiftControlPlanePriorityClass() *schedulingv1.PriorityClass {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: DefaultPriorityClass,
 		},
-		Value:         100000000,
-		GlobalDefault: false,
-		Description:   "This priority class should be used for hypershift control plane pods not critical to serving the API.",
+		Value:            100000000,
+		GlobalDefault:    false,
+		Description:      "This priority class should be used for hypershift control plane pods not critical to serving the API.",
+		PreemptionPolicy: &preemptionPolicy,
```

Comment:
super 10x mega nit: now that we have the generic `ptr.To()` it keeps the value local to where it's used and is easier to read